api/app/tests/apps/signals/test_filters_v1.py

```diff
@@ -0,0 +1,297 @@
+from datetime import datetime, timedelta
```

Comment:
Do we also want to test the actual signals returned by the filter? 

Now we only check if the count is what we want it to be. How do we know that this will also contain the correct signals? Maybe a bit nitpicking here but with dates and times there is a lot of stuff that could potentially go "wrong" but maybe result in a count of items that we expect.

Reply:
Nope. If you can come up with a case that will fail this, I'm happy to further implement the checking of the ID's. ;)

The reasoning behind it: The signals are distributed following a known distribution. All test cases (before, after, between with overlap, disjunct sets) are covered. We know how many results to expect at any given interval. Yes, in theory it may be possible that the 2 items returned aren't the 2 we are expecting (false positive), but in that case I'm sure other tests will fail on that same problem, so the false positive would never go unnoticed. Not checking the ID's would only be an issue if it is somehow possible that _all_ tests succeed while _one or more_ tests return wrong values. In all other cases the problem will come to light while checking the failing tests. In this case I'm confident that if one test inadvertently returns the wrong values other tests will fail.

Reply:
For now this is ok.